### DISCUSSION TITLE
Merge GitLens Inspect with Regular Source Control Pane

### DISCUSSION THREAD

**Discussion by mjmatthiesen** [gitkraken/vscode-gitlens#2747] [Q&A] [answered]
With some recent update, I lost the file history, commit details, and so on, from my source control pane. Instead, I got a new pane. I am looking for a setting to have these back where they belong, the source control pane, but can't find one amongst the thousands of options.

[I know that once upon a time it was a setup option](https://stackoverflow.com/a/64856298/6504973). But I cannot for the life of me figure out where this option disappeared to.

Not very fun to open vs code and have your layout changed because an extension updated.

**Comment by eamodio** [comment, marked as answer]
Views really shouldn't have moved in your existing windows, though it seems like there are some cases where that still happens. Unfortunately extensions get very little control of views, outside of their default locations. 

As for moving the views back, you can drag and drop views freely from/to any location.

**Reply by mjmatthiesen** [reply]
Apparently I am just a dumbo, didn't even realize I could drag and drop. The only real difference I can notice now is that it says "GitLens: File History" instead of just File History, and that may be why.


Thanks!

**Reply by mjmatthiesen** [reply]
Sorry to follow up forever later. But for some reason whenever I open a new project, VS Code auto expands some of the headers.
![image](https://github.com/gitkraken/vscode-gitlens/assets/16603906/9feece07-c0ce-4929-94c4-408098e6a2c9)
Not sure if that's an issue with the extension or vs code. I haven't been able to find any resolution whatsoever.
